letsencrypt/tests/auth_handler_test.py

```diff
@@ -148,6 +152,66 @@ def test_perform_failure(self):
         self.assertRaises(
             errors.AuthorizationError, self.handler.get_authorizations, ["0"])
 
+    @mock.patch("acme.challenges.HTTP01Response.simple_verify")
+    @mock.patch("acme.challenges.TLSSNI01Response.simple_verify")
+    @mock.patch("letsencrypt.auth_handler.logger")
+    def test_simple_verify(self, mock_logger, mock_tlssni01_verify, mock_http01_verify):
```

Comment:
Makes sense :) However,

I'm thinking we should rename ``test_simple_verify`` to ``test_solve_challenges`` and then create a test that sends all those examples to ``_simple_verify`` and asserts the return value.

Afterwards we can possibly alter the ``_solve_challenges`` to test fewer cases, where one of them logs a warning, and another doesn't.

Does it seem logical for you?